Source/DTDLv2/Brick/Point/Sensor/Occupancy-.json

```diff
@@ -3,25 +3,12 @@
   "@type": "Interface",
   "contents": [
     {
-      "@type": "Property",
+      "@type": "Component",
       "displayName": {
         "en": "last known value"
       },
       "name": "lastKnownValue",
-      "schema": {
-        "@type": "Object",
-        "fields": [
-          {
-            "name": "value",
-            "schema": "boolean"
-          },
-          {
-            "name": "timestamp",
-            "schema": "dateTime"
-          }
-        ]
-      },
-      "writable": true
+      "schema": "dtmi:org:w3id:rec:DoubleValueObservation;1"
```

Comment:
Mismatch!? lastKnownValue was boolean, now DoubleValueObservation?

Reply:
Great catch, thanks! This is a bug, I'll address and push updated PR.